AreaSin.java

```diff
@@ -1,13 +1,28 @@
 class AreaSin extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * Calculates triangle's area given SAS
+  * @author: Edric
   */
   
   public void run() {
     
-    // start coding here
-    
+    // declare variables
```

Comment:
include dbl prefix, but all variables are doubles. Therefore, my method is feasible